### ISSUE TITLE
Change field label on page for creating a new gloss

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#634]
The second line on the page says "ID Gloss", but should read "Lemma ID Gloss" (to avoid confusion). Please double-check elsewhere in the interface, that we systematically label this field as "Lemma ID Gloss".

**Comment by vanlummelhuizen** [comment]
@ocrasborn I did it for Add new Sign/Morpheme. In the Minimal pairs panel of a Gloss detail view, should it be Lemma ID Gloss or Annotation ID Gloss? Same for panels "Identical phonology but not marked as homonym" and "Marked as homonym but different phonology"? And in the Relations View?

**Comment by ocrasborn** [comment]
Hi @vanlummelhuizen , in all of those it should be Annotation ID Gloss

**Comment by susanodd** [comment]
There's one in the model.py file in DeletedGlossOrMedia.
Does that require a migration to change the label? I left it as it is.

**Comment by vanlummelhuizen** [comment]
> There's one in the model.py file in DeletedGlossOrMedia.
> Does that require a migration to change the label? I left it as it is.

It would require a migration if the help text of CharField is changed. But I think it is already correct as it is, right?

**Comment by susanodd** [comment]
It's this one inside DeletedGlossOrMedia:
`idgloss = models.CharField("ID Gloss", max_length=50)`
I think that's in the History display of the gloss?
It could show up if Django does the display.

**Comment by vanlummelhuizen** [comment]
Ah, yes. I was looking at the line below that. I think both `save_info_about_deleted_gloss` and `DeletedGlossOrMedia` should be revised. I think the whole gloss/media deletion process could use some reviewing and revision. I will make new issue for that.
